experimenter/tests/integration/nimbus/utils/filter_expression.js

```diff
@@ -1,13 +1,11 @@
-async function remoteSettings(targetingString, recipe) {
-    const { TelemetryEnvironment } = ChromeUtils.importESModule("resource://gre/modules/TelemetryEnvironment.sys.mjs");
-    await TelemetryEnvironment.onInitialized();
-
-    const { ASRouterTargeting } = ChromeUtils.importESModule("resource:///modules/asrouter/ASRouterTargeting.sys.mjs");
-    const { ExperimentAPI } = ChromeUtils.importESModule("resource://nimbus/ExperimentAPI.sys.mjs");
-    const { TargetingContext } = ChromeUtils.importESModule("resource://messaging-system/targeting/Targeting.sys.mjs");
+// Initialize modules once
```

Comment:
Remove this comment. These modules are not being initialized, they are being imported. (a la regular `import` in js)